app/routes.py

```diff
@@ -116,11 +116,8 @@ def verify_pains():
                 found = True
 
             if(not(found)):
-                print('Could not cluster {smile}'.format(smile=smile))
-
-
-
-
+              print('Could not cluster {smile}'.format(smile=smile))
+  
```

Comment:
Could we potentially add the chemicals that fail here to another error banner along the lines of 'we could not cluster this smile'